code/modules/mob/living/carbon/human/human.dm

```diff
@@ -112,6 +115,9 @@
 
 	if(new_species_name)
 		s_tone = random_skin_tone(new_species_name)
+	skin_r = rand(0,255)
```

Comment:
These will be necessary if a human is transformed into a species which has the `MULTICOLOR` flag, and it's easiest to just set the values when the human is created than in every place in which they might change species.

What if I renamed the vars to something like `multicolor_skin_r` and added a comment explaining their use?